jmclient/jmclient/client_protocol.py

```diff
@@ -766,15 +788,15 @@ def start_daemon_on_port(p, f, name, port_offset):
     # starts in jmclient.payjoin:
     if usessl:
         if factory:
-            reactor.connectSSL(host, port, factory, ClientContextFactory())
+                reactor.connectSSL(host, port, factory, ClientContextFactory())
```

Comment:
Is this extra indentation a formatting mix up?

Reply:
Done in https://github.com/abhishek0405/joinmarket-clientserver/commit/828cec29717e88f69cb82c20fc42b3699199d9f8